lib.php

```diff
@@ -32,7 +32,7 @@ function tool_abconfig_after_config() {
         $manager = new tool_abconfig_experiment_manager();
 
         // Check if the param to disable ABconfig is present, if so, exit.
-        if (array_key_exists('abconfig', $_GET) && $_GET['abconfig'] == 'off') {
+        if (optional_param('abconfig', null, PARAM_TEXT) == 'off') {
```

Comment:
why not PARAM_BOOL ?

Reply:
Due to this being manually entered. E.g. abconfig=false or abconfig=0 isn't quite as semantically clear as specifying off to disable in an emergency. I can change it, but thats the reasoning behind off